### DISCUSSION TITLE
Simulate the network failure in the pod, and an error occurs:"unable to set ip tables chains for pod storage-cli-skt2t"

### DISCUSSION THREAD

**Discussion by LZGyx** [chaos-mesh/chaos-mesh#3787] [Q&A]
## Bug Report

**What version of Kubernetes are you using?**

**What version of Chaos Mesh are you using?**
v2.4.1

**What did you do? / Minimal Reproducible Example**
Simulate the network failure in the pod,and an error occurs:"Failed to apply chaos: fialed to apply for pod product-storage/storage-cli-skt2t: unable to set ip tables chains for pod storage-cli-skt2t"
The network-delay.yaml file I configured is as follows：
kind: NetworkChaos
apiVersion: chaos-mesh.org/v1alpha1
metadata:
  namespace: product-storage
  name: pod-net-delay
  annotations:
    experiment.chaos-mesh.org/pause: 'false'
    kubectl.kubernetes.io/last-applied-configuration: >
      {"apiVersion":"chaos-mesh.org/v1alpha1","kind":"NetworkChaos","metadata":{"annotations":{"experiment.chaos-mesh.org/pause":"true"},"name":"pod-net-delay","namespace":"product-storage"},"spec":{"action":"delay","delay":{"correlation":"0","jitter":"0ms","latency":"2ms"},"direction":"to","duration":"4m","mode":"all","selector":{"namespaces":["product-storage"],"pods":{"product-storage":["storage-cli-skt2t"]}}}}
spec:
  selector:
    namespaces:
      - product-storage
    pods:
      product-storage:
        - storage-cli-skt2t
  mode: all
  action: delay
  duration: 4m
  delay:
    latency: 2ms
    correlation: '0'
    jitter: 0ms
  direction: to

**What did you expect to see?**

**What did you see instead?**

**Output of chaosctl**

**Comment by STRRL** [comment]
Hi @LZGyx , could you provide us the log of `chaos-daemon-xxxx` at the on the same worker node.
